circe/src/main/scala/org/http4s/circe/CirceInstances.scala

```diff
@@ -97,7 +101,7 @@ trait CirceInstances extends JawnInstances {
   implicit def jsonEncoder[F[_]]: EntityEncoder[F, Json] =
     jsonEncoderWithPrinter(defaultPrinter)
 
-  private def fromJsonToChunk(printer: Printer)(json: Json): Chunk[Byte] =
+  def fromJsonToChunk(printer: Printer)(json: Json): Chunk[Byte] =
```

Comment:
What necessitated adding this to the public API?

Reply:
@rossabaker sorry, it slipped through. Not intentional. Added back the private and pushed.

Reply:
One more thing that slipped through was the first commit message containing a `WIP` in it. I modified the commit message to remove the `WIP`  and force pushed.